x-pack/legacy/plugins/ml/public/application/jobs/new_job/pages/components/pick_fields_step/components/influencers/influencers.tsx

```diff
@@ -25,6 +25,9 @@ export const Influencers: FC = () => {
   useEffect(() => {
     jobCreator.removeAllInfluencers();
     influencers.forEach(i => jobCreator.addInfluencer(i));
+    if (jobCreator instanceof MultiMetricJobCreator) {
```

Comment:
it's worth noting here that there is a `isMultiMetricJobCreator` type guard function that is typically used in the job wizards for checks like this.
But using `instanceof` is just as good.